source/extensions/filters/http/cache/cache_filter.cc

```diff
@@ -212,9 +212,7 @@ CacheFilter::resolveLookupStatus(absl::optional<CacheEntryStatus> cache_entry_st
       case FilterState::DecodeServingFromCache:
         ABSL_FALLTHROUGH_INTENDED;
       case FilterState::Destroyed:
-        // TODO (capoferro): ENVOY_BUG prevents code coverage from working. When we fix that, we
-        // should convert this to an ENVOY_BUG.
-        ENVOY_LOG(error, absl::StrCat("Unexpected filter state in requestCacheStatus: cache lookup "
+        ENVOY_BUG(false, absl::StrCat("Unexpected filter state in requestCacheStatus: cache lookup "
```

Comment:
you can use IS_ENVOY_BUG for this, right?

Reply:
True, I have updated it to IS_ENVOY_BUG